redaxo/src/core/lib/console/package/list.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @internal
+ */
+class rex_command_package_list extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('List available packages')
+            ->addOption('search', 's', InputOption::VALUE_REQUIRED, 'filter list')
+            ->addOption('installed-only', 'i', InputOption::VALUE_NONE, 'only list installed packages')
+            ->addOption('activated-only', 'a', InputOption::VALUE_NONE, 'only list active packages')
+            ->addOption('using-exit-code', null, InputOption::VALUE_NONE, 'if no package matches your filter the command exits with error-code 1, otherwise with 0')
```

Comment:
Finde ich von der Namensgebung her unpassend

Auch net perfekt, aber meine erste Idee:
```suggestion
            ->addOption('error-when-empty', null, InputOption::VALUE_NONE, 'if no package matches your filter the command exits with error-code 1, otherwise with 0')
```

Reply:
ist besser als meine, also übernommen :wink: